tests/end-to-end/backup/test-cmd/before-backup/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 The Kyma Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM curlimages/curl:7.66.0
+COPY before-backup.test /
+ENTRYPOINT ["/before-backup.test"]
```

Comment:
Please add label to be consistent with the others and also copy the license folder as done in the previous dockerfile.  Please do the same for the other dockerfile as well.

Reply:
sure